task1/src/app/cources/cources.component.html

```diff
@@ -1,4 +1,5 @@
 <div class="page">
-    <app-section></app-section>
-    <app-courceitem (deleteCourceEvent)="deleteCource($event)" [courceItem]="item" *ngFor="let item of cources"></app-courceitem>
- </div>
\ No newline at end of file
+    <app-section (searchCourceEvent)="search($event)"></app-section>
+    <app-courceitem  (deleteCourceEvent)="deleteCource($event)"   [courceItem]="item" *ngFor="let item of cources  | search : 'Title' : searchText | orderby:'-ConductAt'"></app-courceitem>
```

Comment:
orderby dziala dobrze tylko dla typy Data, tak miało by, więc jest OK. Ale skoro tworzysz parametr to znaczy że powinna być możliwość zmiany np na 'Description'. Jak będziesz chciał się pobawić to ok, jak nie to nie - zadanie wykonane poprawnie.